lib/beaker-pe/install/pe_utils.rb

```diff
@@ -450,9 +452,36 @@ def do_install hosts, opts = {}
               else
                 prepare_host_installer_options(host)
                 register_feature_flags!(opts)
-                generate_installer_conf_file_for(host, hosts, opts)
+
+                enterprise_path = '/etc/puppetlabs/enterprise'
```

Comment:
The logic in do_install is already very cluttered. Let's take those whole new if/else block and move it out to a descriptive method taking host, hosts, opt.

Might want to do the same for the scp_from call as well for clarity.